grenache/README.md

```diff
@@ -0,0 +1,417 @@
+# Grenache testnet
+
+The following describes what is required for a single VM that will act as a validator in the `sommelier` grenache testnet. This testnet will involve upgrading from `sommelier` 3.1.1 to 4.0.0, so we will first be installing the earlier binary. We will later be replacing our usage of `gorc` with `steward` as this is the same transition the validators will be going through.
+
+Use a recent Ubuntu image when creating your VM.
+
+## VM Requirements
+
+Compute: 2 CPU
+RAM: 8 GB
+Disk: 50GB-100GB
+Open Ports:
+
+- 26656 (tendermint p2p)
+- 26657 (tendermint rpc)
+- 9090  (tendermint grpc)
+- 1317  (cosmos-sdk api)
+
+## Dependencies
+
+After you spin up your VM with the above specs, install the following dependencies and configure your systemd unit files.
+
+```bash
+sudo apt update -y
+sudo apt upgrade -y
+sudo apt install wget build-essential git nano jq make snapd -y
+
+```
+
+### Install binaries
+
+#### `sommelier`
+
+```bash
+cd ~
+mkdir sommelier_3.1.1_linux_amd64
+cd sommelier_3.1.1_linux_amd64
+wget https://github.com/PeggyJV/sommelier/releases/download/v2.0.0/sommelier_3.1.1_linux_amd64.tar.gz
```

Comment:
Typo here. Believe this should be v3.1.1 (https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz)

Reply:
Yep, fixed it, good catch.